### PR TITLE
0.2/706 revised collection information wording logic

### PR DESCRIPTION
**Pull request by alexalala**
https://trello.com/c/Bi9ojHcL/706-revised-collection-information-wording-logic

🐆

### CODE REVIEW

File: resources/views/service/edit_registration.blade.php [hunk 1]
```diff
@@ -93,22 +93,43 @@
...
+                                {{ $family->entitlement }}
+                            </strong> 
+                            per week
+                        </li>
+                        <li>
+                            Has 
+                            <strong>
+                                {{ count($family->children) }}
+                            </strong> 
+                            children on the scheme
```
Comment (charlesstrange2) [inline resources/views/service/edit_registration.blade.php]: "children registered"?

I think that, technically, kids above 5 years are not "on scheme", becuase they're too old.

**Review by charlesstrange2 — CHANGES_REQUESTED** [review]
one word change :-)

File: resources/views/service/edit_registration.blade.php (hunk 1 above)
Comment (alexalala) [reply resources/views/service/edit_registration.blade.php]: true! Will amend and push up now

**Review by charlesstrange2 — APPROVED** [review]

**Comment by charlesstrange2** [comment]
Groovey :saxophone:

**Pull request merged**